connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -900,7 +900,7 @@ public void testReplicateFromLatest() throws Exception {
 
         String backupTopic = remoteTopicName(topic, PRIMARY_CLUSTER_ALIAS);
         // wait for at least the expected number of records to be replicated to the backup cluster
```

Comment:
from this comment, I think what it wants to do is, verify that there are enough records in primary first, then consume all from backup to check that there are enough records in backup